### PR TITLE
Fix #2488 Required Reformatting for administrator_controls_general_view.xml 

### LINKED ISSUE #2488: Required Reformatting for administrator_controls_general_view.xml
The following files require reformatting
 - layout/administrator_controls_general_view.xml
 - layout-land/administrator_controls_general_view.xml
 - layout-sw600dp/administrator_controls_general_view.xml 

Recommended to set up [Macros](https://github.com/oppia/oppia-android/wiki/Ktlint-Guide#macros) to solve this issue

References
 - https://github.com/oppia/oppia-android/wiki/Coding-style-guide#code-formatting
 - https://github.com/oppia/oppia-android/wiki/Ktlint-Guide#macros

### PR DESCRIPTION
**Pull request by chrk2205**
Fixes #2488 
Reformatted:
layout/administrator_controls_general_view.xml
layout-land/administrator_controls_general_view.xml
layout-sw600dp/administrator_controls_general_view.xml

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -161,4 +177,4 @@
       </indentOptions>
     </codeStyleSettings>
   </code_scheme>
-</component>
+</component>
```
Comment (rt4914) [inline .idea/codeStyles/Project.xml]: Undo these changes.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@chrk2205 Suggested nit.

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (chrk2205) [reply .idea/codeStyles/Project.xml]: sure
thanks

**Comment by rt4914** [comment]
@chrk2205 I will have to close this PR because force-push cannot be merged. We should avoid force push as it tampers ther commit history.

**Comment by rt4914** [comment]
> @chrk2205 I will have to close this PR because force-push cannot be merged. We should avoid force push as it tampers ther commit history.

Reference: 5th Point in https://github.com/oppia/oppia-android/wiki#instructions-for-making-a-code-change
